include/dump_offload.hpp

```diff
@@ -335,8 +335,13 @@ inline void resetHandlers()
             BMCWEB_LOG_DEBUG << "No handler to cleanup";
             return;
         }
-        handler->first->close();
-        BMCWEB_LOG_CRITICAL << "INFO: resetHandlers cleanup";
+        if ((handler->second->dumpType == "system") ||
+            (handler->second->dumpType == "resource"))
+        {
+            handler->first->close();
```

Comment:
Why don't we want this on all dump types like we had before?

Reply:
This is only needed for system/resource dumps
Other dumps like Hostboot, hw dump, and sbe dumps are stored on bmc